module-apps/apps-common/locks/handlers/PhoneLockHandler.cpp

```diff
@@ -486,11 +486,16 @@ namespace locks
         phoneLockTimer.start();
         phoneLockTimeUpdateInfoAction(LockTimerState::Start);
 
+        setNextUnlockAttemptFormattedTime();
+    }
+
+    void PhoneLockHandler::setNextUnlockAttemptFormattedTime() noexcept
+    {
         if (lockedFor < utils::secondsInMinute) {
             lock.setNextUnlockAttemptFormattedTime(std::to_string(lockedFor) +
                                                    utils::translate("phone_lock_blocked_information_seconds"));
         }
-        else if (lockedFor >= utils::secondsInMinute && lockedFor < 2 * utils::secondsInMinute) {
+        else if (lockedFor < 2 * utils::secondsInMinute) {
```

Comment:
why it is done this way instead of duration_cast of 2 minutes? :D

Reply:
Changed to duration_cast ;)